plothist/plotters.py

```diff
@@ -542,15 +537,21 @@ def plot_comparison(
                 lw=0,
             )
         ax.axhline(1, ls="--", lw=1.0, color="black")
-        ax.set_ylabel(r"$\frac{" + x1_label + "}{" + x2_label + "}$")
+        if comparison_ylabel is None:
+            ax.set_ylabel(r"$\frac{" + x1_label + "}{" + x2_label + "}$")
+        else:
+            ax.set_ylabel(comparison_ylabel)
```

Comment:
Just a single `if comparison_ylabel is not None: ax.set_ylabel(comparison_ylabel)` at the end to overwrite